### PR TITLE
Upgrade Maven dependencies

### PR DESCRIPTION
**Pull request by sergeyvladimirovich**
Upgrading to later versions of librato-java and third party Maven dependencies.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -32,7 +32,7 @@
         <tag>HEAD</tag>
     </scm>
     <properties>
-        <dropwizard.metrics.version>3.1.2</dropwizard.metrics.version>
+        <dropwizard.metrics.version>3.2.2</dropwizard.metrics.version>
```
Comment (mheffner) [inline pom.xml]: Is it possible to do this bump as a separate PR/release? If you look at the open branches we've maintained separate branches for older DW metrics versions.

Comment (sergeyvladimirovich) [inline pom.xml]: Sure.
